### DISCUSSION TITLE
Deforum "Generate" button not working in latest commit

### DISCUSSION THREAD

**Discussion by brianjz** [vladmandic/sdnext#1271] [General]
Does anyone else have an issue with Deforum not working. All settings are working, but the "Generate" button is not. I am on the most recent Vlad commit 22f63c42 and the most recent Deforum - f761a2f1.

When I hit Generate, the browser console gives this (Firefox, if that matters):

```
Uncaught (in promise) ReferenceError: showSubmitButtons is not defined  
    submit_deforum deforum.js:3  
    anonymous Blocks-a0ac8a4c.js line 55 > AsyncFunction:3
    je Blocks-a0ac8a4c.js:56
    bo Blocks-a0ac8a4c.js:56
    Fr index.mjs:1170
    Fr index.mjs:1170
    f Button.svelte:12
    Fr index.mjs:1170
    Fr index.mjs:1170
    v Button.svelte:11
```

The showSubmitButtons function exists in the Vlad JS files: https://github.com/vladmandic/automatic/blob/9e66d88e21b79b3126419267022b39aa57f6becd/javascript/ui.js#L120

Just curious if anyone else is having a similar issue before I research where to post an issue.

**Comment by vladmandic** [comment]
actually function `showSubmitButtons` was removed as it didn't do anything anymore. but i guess deforum want to use it (why, i have not idea, but ok). i'll just add back it as a dummy function.

**Reply by brianjz** [reply]
Good to know. I know it worked earlier yesterday and then stopped at some point. It was hard to track down since both got updated. Thanks.
